docs/modules/ROOT/pages/documentation/payara-server/server-configuration/ssl-certificates.adoc

```diff
@@ -16,13 +16,18 @@ To add your SSL Certificate to the Payara Server configuration files, you need:
 - The private key associated with your certificate.
 - The Certificate you received from the Certificate Authority based on your Certificate Signing Request (CSR).
 
-You can find the detailed steps of adding a self-signed certificate into Payara for testing purposes in the blog https://blog.payara.fish/securing-payara-server-with-custom-ssl-certificate[Securing Payara Server with Custom SSL Certificate].
+You can generate a self-signed certificate using the command detailed
+xref:documentation/payara-server/server-configuration/certificate-management.adoc#generate-self-signed-certificate[here]
 
+    asadmin generate-self-signed-certificate --dn "CN=test.payara.fish,IP=127.0.0.1" mydomain_certificate
+
+*Alternatively*::
+You can find the detailed steps of adding a self-signed certificate into Payara for testing purposes using keytool and openssl in the blog https://blog.payara.fish/securing-payara-server-with-custom-ssl-certificate[Securing Payara Server with Custom SSL Certificate].
 The private key can be stored in various formats. We assume that the key is in a format which can be read by the Java key tool program, such as PKCS#8 (file starts with `BEGIN ENCRYPTED PRIVATE KEY`)
 
-=== Combine the Private Key with the Certificate.
+===== Combine the Private Key with the Certificate.
```

Comment:
```suggestion
=== Combine the Private Key with the Certificate.
```
Is this changed by mistake? It should be as it was, one level below "Adding SSL Certificate" and on the same level as "Import into keystore.jks"

Reply:
Nope, it was intentional - I'm trying to make it obvious that this is part of the "Alternatively" section, otherwise it implies that it's a required step

Reply:
OK, but it's not really obvious :-/

I would then delete the "Combine the Private Key with the Certificate." header and just leave everything below "Alternatively:" in plain text:

```
*Alternatively*::
You can find the detailed steps of adding a self-signed certificate into Payara for testing purposes using keytool and openssl in the blog https://blog.payara.fish/securing-payara-server-with-custom-ssl-certificate[Securing Payara Server with Custom SSL Certificate].
The private key can be stored in various formats. We assume that the key is in a format which can be read by the Java key tool program, such as PKCS#8 (file starts with `BEGIN ENCRYPTED PRIVATE KEY`)

In a first step (if following the blog), we combine the Private key and the Certificate we received into a file.  This file can then be used to merge our data into the files used by the Payara installation.
In this step, we also specify an alias name for our key/certificate. This alias name is important as we will use it later to refer to our specific SSL Certificate.

The command to perform this combination is the following with OpenSSL:

     openssl pkcs12 -export -in mycert.crt -inkey mycert.key -out mycert.p12 -name mydomain_certificate

The resulting p12 bundle can then be add to the keystore and trutstore in Payara Server as described in the following sections.
```

I added the last sentence to clarify that the generated certificate still needs to be imported into Payara's stores compared to the generate-self-signed-certificate command which already creates them in Payara's stores.